### PR TITLE
SCC-4025 - Add 1.0 release notes

### PR DESCRIPTION
**Pull request by dgcohen**
This PR adds release notes for the initial public release of the RC refactor.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -5,10 +5,33 @@ All notable changes to this project will be documented in this file.
...
+
+- This version represents the first public release of the research catalog refactor.
+- This public release will expose the new Homepage and Advanced Search page, the Header and Footer, as well as the Feedback form (SCC-3614).
+- Other pages will fall back to their currently live DFE versions.
+
+### Added
+
+- My Account Header (SCC-3924)
+- Holds API routes (SCC-3953)
+- Settings API routes (SCC-3954)
```
Comment (7emansell) [inline CHANGELOG:18]: very minor but also checkout renewal route (SCC-3951)

**Review by 7emansell — APPROVED** [review]

File: CHANGELOG [hunk 2]
```diff
@@ -5,10 +5,34 @@ All notable changes to this project will be documented in this file.
...
+## [1.0.0] 2024-03-04
+
+- This version represents the first public release of the research catalog refactor.
+- This public release will expose the new Homepage and Advanced Search page, the Header and Footer, as well as the Feedback form (SCC-3614).
+- Other pages will fall back to their currently live DFE versions.
+
+### Added
+
+- My Account Header (SCC-3924)
+- Holds API routes (SCC-3953)
```
Comment (EdwinGuzman) [inline CHANGELOG:17]: I'd indent lines 17, 18, and 19 under a "My Account" page/feature heading. The Holds API route is specific to the My Account page and there will be a separate "Holds" api route for bibs. So better to differentiate it.

File: CHANGELOG [hunk 3]
```diff
@@ -5,10 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] 2024-03-04
+
+- This version represents the first public release of the research catalog refactor.
```
Comment (EdwinGuzman) [inline CHANGELOG:10]: Nit: capitalize Research Catalog.

Comment (dgcohen) [reply CHANGELOG:12]: Ended up removing this line and creating a Release Notes header

Comment (EdwinGuzman) [reply CHANGELOG:12]: RC should still be capitalized.

**Pull request merged**